packages/image_picker/image_picker_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.8.8+3
+
+* Updates to a newer version of Pigeon.
```

Comment:
probably worth mentioning the version of pigeon, in case we come across pigeon issues later on

Reply:
👍🏻  Done.